proof-market/application-developers/tutorials/managing-payments.mdx

```diff
@@ -0,0 +1,25 @@
+import ApiSchema from '@theme/ApiSchema';
+
+# Managing payments 
+
+This tutorial details how Proof Market handles payments.
+
+## Attaining contract details
+
+Proof Market exists as a contract that can be called from any application. 
```

Comment:
We should link repo with contract and even blockexplorer with address where the contract is deployed, maybe to put this into sepeare file like env or something like that